package.json

```diff
@@ -168,7 +184,8 @@
         "tinycolor2": "1.4.2",
         "unified": "10.1.2",
         "unist-util-visit": "4.1.2",
-        "xstate": "4.28.1"
+        "xstate": "4.28.1",
+        "zustand": "4.3.3"
```

Comment:
Where do we need Zustand in Fondue? I'm wondering.

Reply:
These are `peerDependencies`, `npm@>7` install them automatically where `pnpm` don't.